### DISCUSSION TITLE
Multiplos certificados

### DISCUSSION THREAD

**Discussion by CalebePrates** [TadaSoftware/PyNFe#337] [Q&A]
Tenho uma dúvida.

Se a empresa tem 2 certificados (empresas do mesmo sócio), podemos sofrer um bloqueio por consultar 20 com 1 certificado e 20 com outro? Esse processo de 1 em 1 hora no caso.

Se sim, qual seria uma solução para aumentar o volume de consultas?

**Comment by G3nilson** [comment]
Olá tudo bem. Como os certificados pertecem a mesma empresa os mesmos podem ser utilizados sem problema algum, o cuidado neste caso é que não sejam feitas muitas requisições simultâneas para não cause uso indevido.
Att;
Genilson.

**Reply by CalebePrates** [reply]
Simultâneo não, mas vai ser logo em seguida. A minha dúvida é, se esse volume de consultas pode gerar um bloqueio por se tratar de uma mesma máquina gerando consultas de 2 certificados. Precisamos fazer mais consultas por hora por causa do volume de notas. Então a minha dúvida é:
Se fizer na mesma hora, 20 consultas com 1 certificado e 20 consultas com outro, existe um risco de bloqueio?

Exemplo:
12:00 fazer 40 consultas (20 com cada certificado na mesma máquina)

**Reply by leogregianin** [reply]
@CalebePrates a regra das consultas é por certificado e não por computador/servidor ou ip. Por exemplo, eu tinha um único servidor em nuvem com mais de 2mil certificados consultando de 1 em 1 hora.

**Reply by CalebePrates** [reply]
Excelente. Muito obrigado @leogregianin
